README.md

```diff
@@ -223,9 +223,9 @@ For each relevant `$di->params['ExampleForAutoResolution']` element that is miss
 
 ```php
 <?php
-$di->params['ExampleForAutoResolution']['foo'] = 'bar';
 $di->params['ExampleForAutoResolution']['baz'] = array();
 $di->params['ExampleForAutoResolution']['dib'] = $di->lazyNew('Example');
+$di->params['ExampleForAutoResolution']['foo'] = 'bar';
```

Comment:
Depends :-) . I agree it is good to be keeping the same order for future verification and easy look ups.